Wangscape/TerrainImages.cpp

```diff
@@ -45,10 +46,8 @@ void TerrainImages::addTerrain(TerrainID tid, std::string image_filename, std::s
         it = source_images.insert({ filename, img }).first;
     }
     sf::Vector2i offset(offset_x, offset_y);
-    sf::Vector2i box(resolution, resolution);
-    offset *= (int)resolution;
```

Comment:
These two lines caused the source tile offset in the source image to be measured in multiples of the resolution instead of in pixels. It didn't seem like a useful restriction; eventually some users may have source images with tiles not aligned to the grid.